### DISCUSSION TITLE
Need disable plugin  postcss-minify-font-values

### DISCUSSION THREAD

**Discussion by imajim** [postcss/postcss#1804] [Q&A]
Hello,

the plugin postcss-minify-font-values convert font-weight. 
How to disabled it ?
i use webpack encore with symfony

thx.

**Comment by ai** [comment]
Show your PostCSS config

**Comment by imajim** [comment]
```
module.exports = {
    plugins: {
        // include whatever plugins you want
        // but make sure you install these via yarn or npm!

        // add browserslist config to package.json (see below)
        autoprefixer: {}
    }
}
```

**Comment by Semigradsky** [comment]
I think this plugin added via `cssnano`:
`css-minimizer-webpack-plugin` -> `cssnano` -> `cssnano-preset-default` -> `postcss-minify-font-values`
You can disable it via `minimizerOptions` - https://github.com/webpack-contrib/css-minimizer-webpack-plugin
In any case this plugin shoudn't be a problem. @imajim why it is a problem for you?

**Comment by imajim** [comment]
I use Mpdf Lib and he doesn't recognize 400, 700, 900 for font-weight.(only accept : normal, bold bolder...)

**Reply by imajim** [reply]
my solution since now i use <strong>
